ci/pipelines/shared/jinja.variables.yml

```diff
@@ -202,3 +202,15 @@ tests:
   PLATFORM: windows
   RAM: '64'
   name: WindowsUnit
+- ARTIFACT_SLUG: apicheck
+  CALL_STACK_TIMEOUT: '20700'
+  CPUS: '4'
+  DUNIT_PARALLEL_FORKS: '0'
+  EXECUTE_TEST_TIMEOUT: 1h
+  GRADLE_TASK: geode-assembly:japicmp
+  MAX_IN_FLIGHT: 1
+  PARALLEL_DUNIT: 'false'
+  PARALLEL_GRADLE: 'false'
+  PLATFORM: linux
+  RAM: '16'
+  name: ApiCheck
```

Comment:
will this result in both ApiCheckOpenJdk8 and ApiCheckOpenJdk11 jobs?  Is there any value to having both?  Ideally we would add a new flag in jinja.variables to allow each job to specify JDK version(s) desired per job.  But that can be a future enhancement.